### PR TITLE
fix(edge-identity/fs-override): fix enabled not being updated correctly 

### PR DESCRIPTION
**Pull request by gagantrivedi**

### CODE REVIEW

**Comment by gagantrivedi** [comment]
Depends on https://github.com/Flagsmith/flagsmith-engine/pull/119

File: api/edge_api/identities/serializers.py [hunk 1]
```diff
@@ -114,12 +114,12 @@ def save(self, **kwargs):
                 raise serializers.ValidationError(
                     "Feature state already exists."
                 ) from e
-        if self.validated_data.get("multivariate_feature_state_values"):
-            self.instance.multivariate_feature_state_values = self.validated_data[
-                "multivariate_feature_state_values"
-            ]
-
         self.instance.set_value(feature_state_value)
+        self.instance.enabled = self.validated_data["enabled"]
```
Comment (matthewelwell) [inline api/edge_api/identities/serializers.py:118]: Do we allow PATCH requests to this endpoint or only PUT? Seems like we might need to disable PATCH for this code to work.

Comment (gagantrivedi) [reply api/edge_api/identities/serializers.py:118]: I am pretty sure we don't do PATCH, but let me ask Kyle before disabling it

Comment (gagantrivedi) [reply api/edge_api/identities/serializers.py:118]: Yeah, we don't use PATCH

**Review by matthewelwell — APPROVED** [review]

**Pull request merged**